funcbench/main.go

```diff
@@ -50,38 +51,49 @@ type benchmarkTester struct {
 func main() {
 	// Show file line with each log.
 	log.SetFlags(log.Ltime | log.Lshortfile)
-	app := kingpin.New(filepath.Base(os.Args[0]), "benchmark result posting and formating tool.\n-i location of github hook file (even.json)")
+	app := kingpin.New(
+		filepath.Base(os.Args[0]),
+		"benchmark result posting and formating tool.",
+	)
 	app.HelpFlag.Short('h')
-	input := app.Flag("input", "path to event.json").Short('i').Default("/github/workflow/event.json").String()
+
+	owner := app.Flag("owner", "github owner or organisation name").Default("prometheus").Short('o').String()
+	repo := app.Flag("repo", "github repository name").Default("prometheus").Short('r').String()
+
+	prNumber := app.Flag("pr", "pull request ID").Short('p').Required().Int()
```

Comment:
Let's not be lazy and name it `pull-request`.

Why we need this though?

Reply:
What about just `initial-branch`? You can put then `pulls/xxx/head` etc..

Reply:
Sorry, what is `initial-branch`? Like in this PR, the original branch name of my fork is `funcbench-args`, so initial-branch is `funcbench-args`? I assume that `xxx` in `pulls/xxx/head` are always digits.

Reply:
Let's merge like this, I will show on my PR what I meant (: